Assets/MixedRealityToolkit.Examples/Demos/HandTracking/CustomProfiles/HandInteractionGestireEventsMixedRealityInputSimulationProfile.asset

```diff
@@ -0,0 +1,54 @@
+%YAML 1.1
```

Comment:
Nit Assets/MixedRealityToolkit.Examples/Demos/HandTracking/CustomProfiles/HandInteractionGestireEventsMixedRealityInputSimulationProfile.asset

I think that Gesture is mispelled in all of these example file names within this folder. (It's spelled Gestire here, but should be Gesture)

Reply:
Well spotted, thanks.